### PR TITLE
Update views.rst

### PR DESCRIPTION
**Pull request by AdrienCS**
French translation of [alternative-control-structures](https://book.cakephp.org/3.0/fr/views.html#alternative-control-structures)

### CODE REVIEW

**Comment by AdrienCS** [comment]
@ravage84 If you could add @HavokInspiration to the reviewers as he was in a previous messy version of this pull-request.

**Comment by ravage84** [comment]
@AdrienCS sure, you are welcome.

File: fr/views.rst [hunk 1]
```diff
@@ -93,18 +93,17 @@ Using Short Tag support::
...
-``endfor``, ``endforeach``, and ``endwhile``. Also notice that instead of using
-a ``semicolon`` after each structure (except the last one), there is a
-``colon``.
-
-The following is an example using ``foreach``:
+Structures de Contrôle Alternatives
+-----------------------------------
+
+Les structures de contrôle tel que ``if``, ``for``, ``foreach``, ``switch``, et ``while``
+peuvent être écrites dans un format simplifié. Remarquez l'absence d'accolades. A la place,
```
Comment (HavokInspiration) [inline fr/views.rst]: "À" (avec l'accent)

File: fr/views.rst [hunk 2]
```diff
@@ -93,18 +93,17 @@ Using Short Tag support::
...
+l'accolade de fin du ``foreach``  est remplacé par ``endforeach``. Chacune des structures de contrôle 
+listées ci-dessous a une syntaxe de fermeture similaire: ``endif``,
+``endfor``, ``endforeach``, et ``endwhile``. Vous remarquerez aussi qu'à la place du ``point-virgule`` après
+chaque structure (à l'exception du dernier), il y a un ``double-point``.
```
Comment (HavokInspiration) [inline fr/views.rst]: "à l'exception de la dernière" peut-être ? Si ça fait référence à "chaque structure".

File: fr/views.rst [hunk 3]
```diff
@@ -93,18 +93,17 @@ Using Short Tag support::
...
-a ``semicolon`` after each structure (except the last one), there is a
-``colon``.
-
-The following is an example using ``foreach``:
+Structures de Contrôle Alternatives
+-----------------------------------
+
+Les structures de contrôle tel que ``if``, ``for``, ``foreach``, ``switch``, et ``while``
+peuvent être écrites dans un format simplifié. Remarquez l'absence d'accolades. A la place,
+l'accolade de fin du ``foreach``  est remplacé par ``endforeach``. Chacune des structures de contrôle
```
Comment (HavokInspiration) [inline fr/views.rst]: "l'accolade [...] est remplacéE"

**Comment by HavokInspiration** [comment]
@AdrienCS Why closing ? This was a perfectly fine PR. You can open it back, make the changes and this will be merged ;)

**Comment by HavokInspiration** [comment]
Thanks a lot @AdrienCS !

**Pull request merged**